admin/access-control/manage.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Authentication management
+description: Learn how to manage Coder authentication.
+---
+
+By default, Coder enables **built-in authentication**, though you can change
+this if desired.
+
+To do so, go to **Manage** > **Users**. Find the user whose authentication type
```

Comment:
nit: here it looks like we're bolding the directions "**Manage** > **Users**"

but below we don't when we say "Under Admin > Manage >
Authentication"

Is this intentional?

Reply:
Unintentional. I try to bold UI instructions to help people skim and find stuff